### PR TITLE
Symantec Management Center Connector v1.0.0 uncertified

### PR DESCRIPTION
**Pull request by cs-omkar-kumbhar**
UTCs:
Connector installation verified.
Connector logo verified.
Actions and Playbook list verified.
Playbook tags verified.
All Playbook are in info mode verified.
All Playbook are in inactive mode verified.

### CODE REVIEW

File: symantec-management-center/playbooks/playbooks.json [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+              "arguments": {
+                "name": "Symantec Management Center",
+                "config": "''",
+                "params": [],
+                "version": "1.0.0",
+                "connector": "symantec-management-center",
+                "operation": "get_policy_content_by_version",
+                "operationTitle": "Get Policy Content By Version",
+                "step_variables": {
+                  "output_data": "{{vars.result}}"
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/playbooks/playbooks.json]: Remove it as it causes duplication of data in the data base from all actions

File: symantec-management-center/operations.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+from requests import request, exceptions as req_exceptions
+from connectors.core.connector import get_logger, ConnectorError
+
+
+logger = get_logger("symantec-management-center")
+
+
+class Symantec:
+    def __init__(self, config, *args, **kwargs):
+        server_url = config.get("server_url")
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/operations.py:17]: strip('/') from the url

File: symantec-management-center/operations.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+    def api_request(self, method, endpoint, params={}, data={}):
+        try:
+            endpoint = self.url + endpoint
+            headers = {
+                "Content-Type": "application/json",
+                "Accept": "application/json"
+            }
+            logger.debug(f"\n-------------req start-------------\n{method} - {endpoint}\nparams: {params}\ndata: {data}\n")
+            response = request(method, endpoint, headers=headers, auth=(self.username, self.password), params=params, data=json.dumps(data), verify=self.verify_ssl)
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/operations.py:33]: Include the curl utils action to make the curl of all the requests.

File: symantec-management-center/info.json [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+          "description": "(Optional) Query filter parameter to filter by description."
+        },
+        {
+          "title": "Name",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "name",
+          "tooltip": "Query filter parameter to filter by author.",
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/info.json]: change it to name in both tooltip and description.

File: symantec-management-center/operations.py [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+def get_policy_details(config, params):
+    ob = Symantec(config)
+    params = build_params(params)
+    return ob.api_request("GET", f"/policies/{params['uuid']}")
+
+
+def create_policy(config, params):
+    ob = Symantec(config)
+    params = build_params(params)
+    return ob.api_request("POST", "/policies", data=params)
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/operations.py:87]: can you take /policies as constant as it is used in all actions?

File: symantec-management-center/info.json [hunk 6]
```diff
@@ -0,0 +1,525 @@
...
+        "revisionInfo": {
+          "revisionNumber": "",
+          "revisionDescription": "",
+          "author": "",
+          "revisionDate": ""
+        }
+      }
+    },
+    {
+      "operation": "get_policy_content_by_version",
```
Comment (cs-utkarsh-surana) [inline symantec-management-center/info.json:485]: Please arrange all the actions in the CRUD sequence.

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**